### PR TITLE
Set up a "Hello world" project with rails-react gem

### PR DESCRIPTION
**Pull request by saadat-ali-awan**
### Project requirements

 - Create a new Rails app called 'hello-rails-react'.
 - Initialize your project with Git.
 - Set up your project with `webpacker` the `react-rails` gems as describe in the previous lesson's tutorial.
 - Make sure that your project has a Postgres database set up. 
 - Create a table for storing your messages and create 5 different greetings. 
 - Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
 - Create a static view that will be the root of your app.

---------

- Create the `App` component with react-router and render it in your static view.
- Create the `Greeting` component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
- Display the random greeting in your `Greeting` component.

### CODE REVIEW

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Saadat123456, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
- [ ] Kindly fix this error
image
I have followed the steps provided in the README and I'm getting this error.



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @Saadat123456, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

You have submitted a project, that Linters check are not running in the GitHub/action
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/static_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class StaticController < ApplicationController
+  def index; end
+end
```
Comment (BenMukebo) [inline app/controllers/static_controller.rb:3]: - [ ] Kindly as requested by the previous reviewer, make sure your app runs correctly on your machine. 👍🏻 
- 🤔 I have followed too the steps provided in the README and I'm still getting the same error. Please solve this 😄 

rails_react

Comment (BenMukebo) [reply app/controllers/static_controller.rb:3]: - [ ] Hello !!! After testing the project several times with the student's computer, I can confirm that the project runs perfectly without any errors, as you can see in this screenshot 👇🏻

Screenshot taken on 2022-07-13 at 00-11-23 (1)

- Please approve this!!

**Review by KDlamini — APPROVED** [review]
### Hi @Saadat123456  🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the project requirements. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 
✔️  Correct PR title
✔️  Good PR description
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @Saadat123456 👋,

_This is an additional review as a result of the second opinion request._

I can see that your project is already approved. Am happy to see that the problem has been resolved and that you have completed your project.

Keep rocking 🚀!

**Pull request merged**